Hello/vcpkg-configuration.json

```diff
@@ -0,0 +1,25 @@
+{
+    "registries": [
+      {
+        "kind": "artifact",
+        "location": "https://aka.ms/vcpkg-ce-default",
+        "name": "microsoft"
+      },
+      {
+        "kind": "artifact",
+        "location": "https://artifacts.keil.arm.com/vcpkg-ce-registry/registry.zip",
```

Comment:
@jkrech, shall we point this to the "new" URL already?

Reply:
I don't have a strong opinion. Since the above location needs to be maintained anyways.